include/seqan3/alignment/pairwise/detail/type_traits.hpp

```diff
@@ -83,6 +85,21 @@ template <typename configuration_t>
 struct alignment_configuration_traits
 {
 private:
+    /*!\brief An index type (i.e. unsigned integral) for a score_type which has the same bit size.
+     * \tparam score_t The score type for which the corresponding index type shall be returned; must mode
+     *                 seqan3::arithmetic
+     *
+     * \details
+     *
+     * If the score type models std::integral it will simply be converted to its unsigned counterpart using the
+     * transformation trait std::make_unsigned. For floating point types the bit size is determined and the
+     * corresponding minimal viable unsigned integral type (seqan3::detail::min_viable_uint_t) is returned.
```

Comment:
```suggestion
     * If the score type models std::integral it will simply be converted to its unsigned counterpart.
     * For floating point types the bit size is determined and the corresponding minimal viable
     * unsigned integral type (seqan3::detail::min_viable_uint_t) is returned.
```